app/containers/Certifier/CertificationRequestsContainer.tsx

```diff
@@ -0,0 +1,183 @@
+import React, {useEffect, useState} from 'react';
+import {Button, Alert} from 'react-bootstrap';
+import moment from 'moment-timezone';
+import Link from 'next/link';
+import {graphql, createRefetchContainer, RelayRefetchProp} from 'react-relay';
+import SearchTableLayout from 'components/SearchTableLayout';
+import PaginationBar from 'components/PaginationBar';
+import {CertificationRequestsContainer_query} from '__generated__/CertificationRequestsContainer_query.graphql';
+
+const TIME_ZONE = 'America/Vancouver';
+
+function formatListViewDate(date) {
+  return date ? moment.tz(date, TIME_ZONE).format('MMM D, YYYY') : '';
+}
+
+interface Props {
+  direction: string;
+  orderByField: string;
+  searchField: string[];
+  searchValue: string[];
+  offsetValue: number;
+  handleEvent: (...args: any[]) => void;
+  relay: RelayRefetchProp;
+  query: CertificationRequestsContainer_query;
+}
+
+export const CertificationRequestsComponent: React.FunctionComponent<Props> = ({
+  direction,
+  orderByField,
+  searchField,
+  searchValue,
+  handleEvent,
+  relay,
+  query
+}) => {
+  const [offsetValue, setOffset] = useState(0);
+  const [activePage, setActivePage] = useState(1);
+  useEffect(() => {
+    const refetchVariables = {
+      searchField,
+      searchValue,
+      orderByField,
+      direction,
+      offsetValue
+    };
+    relay.refetch(refetchVariables);
+  });
+
+  const displayNameToColumnNameMap = {
+    Facility: 'facility_name',
+    Organisation: 'operator_name',
+    Status: 'application_revision_status',
+    'Certified By': 'certified_by',
+    'Date Certified': 'certified_at',
+    '': null
+  };
+
+  const body = (
+    <tbody>
+      {query.searchCertificationRequests.edges.map(({node}) => {
+        const status = node.applicationRevisionStatus;
+
+        const facility = node.facilityName;
+
+        const organisation = node.operatorName;
+
+        const certifierName =
+          node.certifiedByFirstName || node.certifiedByLastName
+            ? `${node.certifiedByFirstName} ${node.certifiedByLastName}`
+            : '';
+
+        const {applicationId} = node;
+        const {versionNumber} = node;
+
+        return (
+          <tr key={node.rowId}>
+            <td>{facility}</td>
+            <td>{organisation}</td>
+            <td>{status}</td>
+            <td>{certifierName}</td>
+            <td>{formatListViewDate(node.certifiedAt)}</td>
+            <td>
+              <Link
+                href={`/certifier/certify?applicationId=${applicationId}&version=${versionNumber}`}
+              >
+                <Button className="w-100">View</Button>
```

Comment:
This `applicationId` appears to be the "short" id, whereas you need the "long" one in order to construct a valid URL to the certify page. Can you get that from your search function? Try clicking on the link here.